pingsrv.go

```diff
@@ -149,11 +149,16 @@ func grpcHealthCheck(serverAddr string, svcname string, n int) {
 }
 
 func grpcClient() {
-	if len(flag.Args()) != 1 {
-		usage("Error: fortio grpcping needs host argument")
+	if len(flag.Args()) < 1 || len(flag.Args()) > 2 {
+		usage("Error: fortio grpcping needs either host or host and port arguments. ",
+			"Fortio's default gRPC server port (8079) is used if port is omitted.")
```

Comment:
here it should be %v and *grpcPortFlag instead of 8079

also didn't we say I could put `host:port` (1 argument ? we can also do 2 args as extra flexibility if you want)

Reply:
I'll update the error msg.

When you asked that GRPCDestination take string & port as args, I thought you were looking for separate host/port flags.

Reply:
nah sorry i wasn’t clear, I meant the flag be the port if the port isn’t in the destination already

Reply:
I'm working through the changes and I think I'm about ready to puth the commit. I am sticking with the single `grpcping` flag representing a dest host. The dest can be `hostname`, `ip`, `ip:port` or `hostname:port`. If hostname/ip are specified, then `:*grpcDefaultPort` will be added.